src/spac/spatial_analysis.py

```diff
@@ -27,15 +31,24 @@ def spatial_interaction(
             The analysis method to use, currently available:
             "Neighborhood Enrichment" and "Cluster Interaction Matrix".
 
+        stratify_by : str or list of strs
+            The annotation[s] to stratify the dataset. If single annotation is
+            passed, the dataset will stratify by the unique values in
+            the annotation column. If n (n>=2) annotations are passed,
+            the function will stratify the dataset basing on existing
+            label combinations.

```

Comment:
@ruiheesi, would you please reward to: "The annotation[s] to stratify the dataset when generating interaction plots. If single annotation is passed, the dataset will be stratified by the unique labels in the annotation column. If n (n>=2) annotations are passed, the function will be stratified based on existing combination of labels in the passed annotations."